cliconfig/config.go

```diff
@@ -15,19 +14,18 @@ import (
 )
 
 const (
-	// Where we store the config file
-	CONFIGFILE     = "config.json"
-	OLD_CONFIGFILE = ".dockercfg"
+	// ConfigFile is the name of config file
```

Comment:
@MHBauer this is incorrect, it should be `ConfigFileName`. And please add `cliconfig` to the list in `./hack/make/validate-lint`. You can send a new PR if you want :)